src/schema/v2/startIdentityVerificationMutation.ts

```diff
@@ -77,9 +77,7 @@ export const startIdentityVerificationMutation = mutationWithClientMutationId<
     { identityVerificationId },
     { startIdentityVerificationLoader }
   ) => {
-    if (!startIdentityVerificationLoader) {
-      throw new Error("You need to be signed in to perform this action")
-    }
+    if (!startIdentityVerificationLoader) return null
```

Comment:
You can likely drop this check altogether, there shouldn't ever be a reason this loader would not exist (since it's defined in the without authentication file). We usually don't check presence of loader unless there's a friendlier experience (better error message, etc.) we want for the logged-in only case.

Reply:
Drive by comment. Thanks for this explanation. I wasn't totally clear on the purpose of this check when doing related work. Good to know!